### PR TITLE
active-nav-modification

### PR DESCRIPTION
**Pull request by atolz**
Activate nav active-section

### CODE REVIEW

**Comment by atolz** [comment]
Activate nav active-section

**Review by atolz — COMMENTED** [review]
LGTM

File: portfolio - Copy/index.html [hunk 1]
```diff
@@ -1,109 +1,106 @@
...
+      <div class="contact-container">
+        <h2>Let's make something together.</h2>
+        <div class="social">
+          <a href="http://www.linkedin.com/in/victor-atolagbe" target="_blank" class="soc-link"> LinkedIn</a>
+          <a href="mailto:atolagbevictor.oo@gmail.com" target="_blank" class="soc-link"> atolagbevictor.oo@gmail.com</a>
         </div>
+      </div>
     </section>
 
     <script>
```
Comment (atolzvt) [inline portfolio - Copy/index.html:91]: won't it be better to move the sidebar script to the JS folder too?

**Review by atolzvt — APPROVED** [review]

File: portfolio - Copy/index.html (hunk 1 above)
Comment (atolz) [reply portfolio - Copy/index.html:91]: Hmm, what side
bar script?

Comment (atolzvt) [reply portfolio - Copy/index.html:91]: line 91 of index.html...

**Pull request merged**